views/apps_list.pug

```diff
@@ -18,6 +18,10 @@ block content
             span(aria-hidden='true') ×
           p= message
 
+  - if(apps.length === 0)
+    p= "No rules created yet."
+    p= "They can be created by WHEN type commands."
```

Comment:
It would be better to wrap these messages in `_()`, which marks the strings for translation using gettext.